scripts/qa_crowbarsetup.sh

```diff
@@ -1288,9 +1292,8 @@ EOF
         elif iscloudver 6plus; then
             if [ -n "$want_sles12sp1" ] ; then
                 add_ha12sp1_repo
-            else
-                add_ha12_repo
             fi
+            add_ha12_repo
```

Comment:
i had the issue with hacloud and ceph enabled.. since we need to deploy sp0 for SES2, we also needed ha for sp0..